roles/compute_resources/README.md

```diff
@@ -21,7 +21,7 @@ The following fields are optional and will be omitted by default:
 Example Playbooks
 -----------------
 
-Create a lifecycle environment path with three environments: Library -> Dev -> Test -> Prod
+Create a compute resource for vSphere.
```

Comment:
It handles a list of cr right?

Reply:
Yes, but the example playbook creates just a single one. Should I add more items to the list in the example?

Reply:
Right. Sorry. In that case, no.